src/components/Employee/Compensation/Compensation.tsx

```diff
@@ -31,9 +36,13 @@ import {
 } from '@/api/queries'
 import { RequireAtLeastOne } from '@/types/Helpers'
 
-export type CompensationDefaultValues = RequireAtLeastOne<
-  Partial<Pick<Schemas['Job'], 'rate' | 'title' | 'payment_unit'>>
->
+export type CompensationDefaultValues = RequireAtLeastOne<{
+  rate?: Schemas['Job']['rate']
+  title?: Schemas['Job']['title']
+  payment_unit?: (typeof PAY_PERIODS)[keyof typeof PAY_PERIODS]
```

Comment:
Update to limit payment_unit default value to what we actually have available from pre selected options so when a default value gets supplied this doesn't get out of sync with our infrastructure